syntax = "proto3";

package sim;

message HelloRequest { string name = 1; }
message HelloResponse { string replay = 1; }
service Greeter {
  // send a hello and return a replay
  rpc SayHello(HelloRequest) returns(HelloResponse) {}
}